flutter_vlc_player/example/lib/video_data.dart

```diff
@@ -1,3 +1,4 @@
+// ignore: prefer-match-file-name
```

Comment:
Rename file instead?

Reply:
Renamed all except:
- hardware_acceleration.dart file, there is `HwAcc` class, not sure that we need to rename this class or file. Because class name have abbreviation and file name have description. What do you think? Should we rename this class or file name?

- messages.dart file. There are a lot classes, that connected with message and API. Maybe, better separate this file to some another, and leave only exports in this file. What do you think?